package.json

```diff
@@ -36,13 +36,12 @@
   "devDependencies": {
     "eslint": "^8.2.0",
     "eslint-config-semistandard": "^16.0.0",
-    "eslint-config-standard": "^15.0.1",
+    "eslint-config-standard": "^16.0.3",
```

Comment:
> npm WARN eslint-config-semistandard@16.0.0 requires a peer of eslint-config-standard@>=16.0.3 but none is installed

In my local env I used `npm install` with a brand new `node_modules` folder and I've not met this warning, might be because this version update has happened.

Reply:
Also not found this warning in the CI run: https://cloudant-sdks-jenkins.swg-devops.com/blue/rest/organizations/jenkins/pipelines/Main/pipelines/couchbackup/branches/lint-fixes/runs/8/nodes/25/steps/65/log/?start=0

Reply:
OK thanks, I don't see any peer dep warnings now either.